README.md

```diff
@@ -7,68 +7,172 @@ PADD (formerly Chronometer2) is a more expansive version of the original chronom
 ***Note:** PADD has been adopted by the Pi-hole team, thanks to JPMCK for creating this helpful tool!
 
 ## Setup PADD
-*More in-depth information about setting up PADD can be found in this repo’s [wiki](https://github.com/pi-hole/PADD/wiki/Setup).*
 
 - Get a copy of PADD by running:
 
-```bash
-cd ~
-wget -O padd.sh https://install.padd.sh
-```
-or
-```bash
-cd ~
-curl -sSL https://install.padd.sh -o padd.sh
-```
+  ```bash
+  cd ~
+  wget -O padd.sh https://install.padd.sh
+  ```
+
+  or
+
+  ```bash
+  cd ~
+  curl -sSL https://install.padd.sh -o padd.sh
+  ```
 
 - Make PADD executable by running
 
-```bash
-sudo chmod +x padd.sh
-```
+  ```bash
+  sudo chmod +x padd.sh
+  ```
 
-- Set PADD to auto run by adding the following to the end of `~/.bashrc`:
+## Using PADD
 
-```bash
-# Run PADD
-# If we’re on the PiTFT screen (ssh is xterm)
-if [ "$TERM" == "linux" ] ; then
-  while :
-  do
-    ./padd.sh
-    sleep 1
-  done
-fi
-```
+### PADD on Pi-hole machine
 
-One line version
+- Just run
 
-```bash
-cd ~ ; echo "if [ \"\$TERM\" == \"linux\" ] ; then\n  while :\n  do\n    ./padd.sh\n    sleep 1\n  done\nfi" | tee ~/.bashrc -a
-```
+  ```bash
+  ./padd.sh
+  ```
+
+### PADD from other machine
+
+With PADD v4.0.0 and Pi-hole v6 it is also possible to run PADD from a machine that is not running Pi-hole
+
+  ```bash
+  ./padd.sh --server <DOMAIN|IP>
+  ```
+
+### Authentication
+
+Pi-hole v6 uses a completely new API with a new authentication mechanism
+
+If you run PADD on the same machine as Pi-hole, it's possible to bypass authentication when your local user is member of the `pihole` group (specifically, if you can access `/etc/pihole/cli_password).
```

Comment:
actual location is

`/etc/pihole/cli_pw`

Reply:
Thanks.

Reply:
https://github.com/pi-hole/PADD/pull/416/commits/a35806ca42e7e71a7f8785f6546e6456d7f8ded5